### PR TITLE
Add monitor keyword argument parameter

### PR DESCRIPTION
**Pull request by araffin**
## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

See https://github.com/DLR-RM/rl-baselines3-zoo/issues/247

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist:

- [x] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [x] I have updated the changelog accordingly (**required**).
- [x] My change requires a change to the documentation.
- [x] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [x] I have updated the documentation accordingly.
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [x] I have ensured `make pytest` and `make type` both pass. (**required**)

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

**Comment by qgallouedec** [comment]
Is there a reason why this option is not available on the command line?

**Comment by araffin** [comment]
it is, via the params option (see test), or you meant even easier access?

File: README.md [hunk 1]
```diff
@@ -296,6 +296,16 @@ env_wrapper:
...
+By default, the environment is wrapped with a `Monitor` wrapper to record episode statistics.
+You can specify arguments to it using `monitor_kwargs` parameter to log additional data that *must* be present
+in the info dictionary.
+For instance, for recording success with goal envs (e.g. `FetchReach-v1`):
+
+```yaml
+monitor_kwargs: dict(info_keywords=('is_success',))
+```
+
+
```
Comment (qgallouedec) [inline README.md:308]: I think it could be written more clearly that only the info from the last interaction of the episode is monitored.

Comment (qgallouedec) [reply README.md:308]: Something like:


You can specify arguments to it using `monitor_kwargs` parameter to record additional data that *must* be present in the information dictionary of the last interaction of the episode.
For instance, for recording final x position with `Ant-v3`:

 ```yaml
 monitor_kwargs: dict(info_keywords=('x_position',))
 ```

Comment (qgallouedec) [reply README.md:308]: (For FetchReach, `info_keywords=('is_success',)` is already the default value)

**Review by qgallouedec — APPROVED** [review]

**Pull request merged**